services/net/transcription/TranscriptionManager.cs

```diff
@@ -327,5 +339,49 @@ private async Task<string> RequestTranscriptionAsync(byte[] data, string languag
 
         return sb.ToString();
     }
+
+    /// <summary>
+    /// Check if the clip file contains a video stream.
+    /// </summary>
+    /// <param name="file"></param>
+    /// <returns></returns>
+    /// The same as Capture and Clip services, to be re-factored
+    private async Task<bool> IsVideoAsync(string file)
+    {
+        var process = new System.Diagnostics.Process();
+        process.StartInfo.Verb = $"Stream Type";
+        process.StartInfo.FileName = "/bin/sh";
+        process.StartInfo.Arguments = $"-c \"ffmpeg -i {file} 2>&1 | grep Video | awk '{{print $0}}' | tr -d ,\"";
+        process.StartInfo.UseShellExecute = false;
+        process.StartInfo.RedirectStandardOutput = true;
+        process.StartInfo.CreateNoWindow = true;
+        process.Start();
+
+        var output = await process.StandardOutput.ReadToEndAsync();
+        await process.WaitForExitAsync();
+        return !String.IsNullOrWhiteSpace(output);
+    }
+
+    /// <summary>
+    /// video to audio
+    /// </summary>
+    /// <param name="file">video file</param>
+    /// <returns></returns>
+    private async Task<bool> Video2Audio(string srcFile, string destFile)
+    {
+
+        var process = new System.Diagnostics.Process();
+        process.StartInfo.Verb = $"Stream Type";
+        process.StartInfo.FileName = "/bin/sh";
+        process.StartInfo.Arguments = $"-c \"ffmpeg -i {srcFile} -y {destFile} \"";
+        process.StartInfo.UseShellExecute = false;
+        process.StartInfo.RedirectStandardOutput = true;
+        process.StartInfo.CreateNoWindow = true;
+        process.Start();
+
+        var output = await process.StandardOutput.ReadToEndAsync();
+        await process.WaitForExitAsync();
+        return !String.IsNullOrWhiteSpace(output);
```

Comment:
Not sure returning a boolean value makes a lot of sense here.  Return the full path to the new file.
If the process fails, you need to capture the error and propagating it.